website/index.html

```diff
@@ -115,7 +115,7 @@ <h2 id="example">Example <a href="#example">#</a></h2>
 
       <p>Or a more complex one:</p>
 
-      <pre><code class="typescript language-typescript">import { serve } from "<a href="https://deno.land/std@v0.12/http/server.ts">https://deno.land/std@v0.12/http/server.ts</a>";
+      <pre><code class="typescript language-typescript">import { serve } from "<a href="https://deno.land/std@v0.19.0/http/server.ts">https://deno.land/std@v0.19.0/http/server.ts</a>";
```

Comment:
Couldn't it just be the latest release `https://deno.land/std/http/server.ts`?

Reply:
@nayeemrmn that URL isn't the latest release, though. Unless I'm mistaken it's the latest commit to `master`. Subtle but important difference, especially for `std` IMO because `std` is a collection of modules, and so any PRs that land would have to leave `master` in a consistent state. Incremental changes that break compatibility temporarily between releases would no longer be possible.

One solution to that would be to use a `dev` branch for incremental work. Not sure how people feel about that.